cfp.md

```diff
@@ -34,18 +34,62 @@ Contributions may overlap with any of the following areas:
 
 ### Submission Guidelines
-TBA
+Papers submitted to TCRS '24 must follow the [IEEE ESL Paper Submission Instructions](https://ieee-ceda.org/publication/esl/esl-paper-submission).
+Submitted manuscripts must be **four pages or fewer**, including all figures, tables, and references.
+Submissions exceeding this length will be returned without review. Papers should use 7.875 in x 10.75 in (20 cm x 27.30 cm) trim size and the IEEE transactions two-column format in 10-pt. font.
+The review process is **blind** (not double-blind);
+please do not submit anonymized manuscripts.
+
+### Submission Site
+Please submit your paper at the [Manuscript Central Submission Site](https://mc.manuscriptcentral.com/les-ieee).
+Click on "**Author**," create a new manuscript, and select "**Type**" as "**SI:TCRS24**".
+
+### Conflicts of Interest Requirements
+In addition to the manuscript, the authors are required to submit a **Cover Letter** indicating the **Conflicts of Interest (COIs)** with the program committee members through Manuscript Central.
+The authors can submit the cover letter during the Submission **Step 6: Details & Comments**, using a textbox under **Write Cover Letter**.
+Even if the authors do not have any COIs, they are still required to submit a cover letter confirming that they have no COIs.
+
+### Acceptance Tiers
+At least one of the authors for each paper must fully register and present the accepted paper at the workshop.
+TCRS '24 will have two tiers of final acceptance.
+
+1. *Present-only* acceptance.
+2. *Present-and-publish* acceptance.
+
+The *Present-only* tier acceptance will be given to the submissions that meet the TCRS workshop quality requirements.
+The *Present-and-publish* tier acceptance will be given to the submissions that meet the quality requirements for IEEE ELS journal publication as well as the quality requirements for the TCRS workshop.
 
-### Submission site
+All papers with any of these two tiers of acceptance will be presented at the workshop and distributed as proceedings of the workshop.
+However, only the papers with the *Present-and-publish* tier acceptance will be published in the IEEE ESL special issue of TCRS.
 
-TBA
+### Review Process
+Authors of the submissions will be notified with the first round of reviews.
+Below are the subsequent processes for accepted papers.
+
+1. Authors of the submissions with *Present-only* acceptance will be asked to submit the final PDFs that will be included in the "*proceedings to be distributed*."
+The review result from IEEE ESL will be a "reject" for the submission with *Present-only* acceptance; however, the authors will receive a separate notice for *Present-only* acceptance.
+
+2. Authors of the submissions with *Present-and-publish* acceptance will receive a "minor revision" decision from IEEE ESL and will be required to submit the final camera-ready PDF and a video graphical abstract which is a short video presenting the paper.
```

Comment:
```suggestion
2. Authors of the submissions with *Present-and-publish* acceptance will be expected to incorporate the review feedback into a minor revision that is to be submitted for publication to IEEE ESL. In addition to submitting the final camera-ready version of their paper, authors are asked to submit a video graphical abstract (which is a short video presenting the paper).
```

Reply:
I think the first part of your suggested change is not accurate. They will indeed receive a "minor revision" decision from the journal. Let me come up with a better wording...

Reply:
Yes, I think it matters to avoid confusion. For example, if authors receive "reject" notice they may think they won't be able to present their paper at the workshop either.